### PR TITLE
make more site-specific items configurable

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -20,25 +20,42 @@ box_size: '8x12 Storage (scannable)'
...
 # samples 4 and 5 are autofilled based on incrementing pattern from 2 and 3
+plain_sample6_alias: 'PROJ_S06_0001'
 
-# values for Ocford Nanopore tutorial
+# values only affecting detailed sample mode
+identity_class: 'Identity'
+tissue_class: 'Tissue'
+slide_class: 'Slide'
+stock_class: 'gDNA (stock)'
+alquot_class: 'gDNA (aliquot)'
```
Comment (avarsava) [inline _config.yml:43]: alquot -> aliquot?

File: tutorial-detailed-samples.md [hunk 2]
```diff
@@ -401,23 +376,23 @@ _Edit Sample_ page for a single sample, click the sample's name or alias on the
 To bulk edit a set of samples, click the checkboxes next to them on the _Samples_ page and click
 _Edit_ at the top left of the table.
 
-For attributes which cannot be changed via MISO (e.g. Sample Class), please email gsi@oicr.on.ca
-or file a JIRA ticket in GSI Common to get assistance from the MISO team.
+For attributes which cannot be changed via MISO (e.g. Sample Class), {{ site.miso_admin_contact }} for help. This may
```
Comment (avarsava) [inline tutorial-detailed-samples.md:379]: Does site.miso_admin_contact include a verb or would this evaluate to 'For attributes which cannot be changed via MISO, [email address] for help'?

File: _config.yml (hunk 1 above)
Comment (djcooke) [reply _config.yml]: yep! fixed

File: tutorial-detailed-samples.md (hunk 2 above)
Comment (djcooke) [reply tutorial-detailed-samples.md:379]: Includes a verb. See `_config.yml`: "email gsi@oicr.on.ca or file a JIRA ticket in _GSI Common_"

**Pull request merged**